docs/rules/prefer-node-builtin-imports.md

```diff
@@ -0,0 +1,39 @@
+# import/prefer-node-builtin-imports
```

Comment:
I’m not interested in a rule that solely pushes using the protocol - if anything, it’d be a rule that can be configured to require, or forbid, the protocol.

Reply:
@ljharb What config option should be from user's perspective. Just a boolean to toggle this? If yes then I don't see how is it different from simply have it `on` or `off`

Reply:
"off" is the current state - where you can have `import 'fs'` and `import 'node:path'` in the same file. One rule setting would error on the first one, and require `node:fs`. The other would error on the second one, and require `path`.